src/Catalyst.Core.Modules.P2P.Discovery.Hastings.Tests/UnitTests/DiscoveryTestBuilder.cs

```diff
@@ -212,10 +214,7 @@ public DiscoveryTestBuilder WithBurn(int burnInPeriod = 0)
             PeerId peer = default,
             INeighbours neighbours = default)
         {
-            if (_careTaker == null)
-            {
-                WithCareTaker();
-            }
+            if (_careTaker == null) WithCareTaker();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)